src/navigator/types.tsx

```diff
@@ -265,15 +265,15 @@ export type StackParamList = {
     sendAmount: string
     tokenId: string
   }
-  [Screens.Settings]: { promptConfirmRemovalModal?: boolean; isTabNav?: boolean } | undefined
-  [Screens.SettingsDrawer]: { promptConfirmRemovalModal?: boolean; isTabNav?: boolean } | undefined
+  [Screens.Settings]: { promptConfirmRemovalModal?: boolean } | undefined
+  [Screens.SettingsDrawer]: { promptConfirmRemovalModal?: boolean } | undefined
   [Screens.SignInWithEmail]: {
     keylessBackupFlow: KeylessBackupFlow
   }
   [Screens.Spend]: undefined
   [Screens.StoreWipeRecoveryScreen]: undefined
-  [Screens.Support]: { isTabNav?: boolean } | undefined
-  [Screens.SupportDrawer]: { isTabNav?: boolean } | undefined
+  [Screens.Support]: {} | undefined
```

Comment:
having `{} | undefined` feels janky but I needed it to avoid a build error, is there a nicer way to do this (i.e. to have `undefined` but still be able to use `route` in the component)?

Reply:
The build error you're referring to is the one on DrawerNavigator right? I think we've had similar issues with other components, see other drawer screens that have a @ts-ignore. I think you can just keep this type in the SupportDrawer or add ts-ignore, as we'll be cleaning up the drawer navigator soon anyway

Reply:
would be good to updated type of `Support` to just be `undefined` and keep the weird `{} |` for SettingsDrawer right now